core/utils.h

```diff
@@ -1223,7 +1226,7 @@ bitmap_check_consistency(bitmap_t b, uint bitmap_size, uint expect_free);
 #    endif /* INTERNAL */
 #    define THREAD          \
         ((dcontext == NULL) \
-             ? INVALID_FILE \
```

Comment:
Are we concerned about any case where dcontext isn't present but we wrongly assume that it's because DR is detaching?

Reply:
It could be a `native_exec_list` library who created a new thread or something, so it does not have to be a detach (though in today's most common uses detach is most frequent) but it seems to me that printing extra log statements from any native thread into the global log file is always better than having log statements dropped on the ground: these are verbose messages for diagnostics after all.

Reply:
Sure, makes sense.